internal/sdkv2provider/schema_cloudflare_certificate_pack.go

```diff
@@ -50,7 +50,7 @@ func resourceCloudflareCertificatePackSchema() map[string]*schema.Schema {
 			Type:         schema.TypeString,
 			Required:     true,
 			ForceNew:     true,
-			ValidateFunc: validation.StringInSlice([]string{"digicert", "lets_encrypt", "google"}, false),
+			ValidateFunc: validation.StringInSlice([]string{"digicert", "lets_encrypt", "google", ssl_com"}, false),
 			Default:      nil,
 			Description:  fmt.Sprintf("Which certificate authority to issue the certificate pack. %s", renderAvailableDocumentationValuesStringSlice([]string{"digicert", "lets_encrypt", "google"})),
```

Comment:
we also should put the value here in the description and run the docs generation to pick up on it.

Reply:
yep, that's fine